src/backend/chat/commands/builtin/quotes.ts

```diff
@@ -305,7 +305,7 @@ export const QuotesManagementSystemCommand: SystemCommand<{
                         return resolve();
                     }
 
-                    await twitchChat.sendChatMessage(`Sorry! Couldnt find a quote with that id number.`);
+                    await twitchChat.sendChatMessage(`Sorry! couldn't find a quote with that id number.`);
```

Comment:
`couldn't` should be `Couldn't` here

Reply:
updated Should that also be We couldn't since all others start that way?